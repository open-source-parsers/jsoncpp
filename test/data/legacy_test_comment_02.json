{
   /* C-style comment

    C-style-2 comment */
   "c-test" : {
      "a" : 1,
      /* Internal comment c-style */
      "b" : 2
   },
   // C++-style comment
   "cpp-test" : {
      // Multiline comment cpp-style
      // Second line
      "c" : 3,
      // Comment before double
      "d" : 4.1,
      // Comment before string
      "e" : "e-string",
      // Comment before true
      "f" : true,
      // Comment before false
      "g" : false,
      // Comment before null
      "h" : null
   }
}
