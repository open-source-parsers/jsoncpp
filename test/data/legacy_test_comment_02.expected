.={}
/* C-style comment

    C-style-2 comment */
.c-test={}
.c-test.a=1
/* Internal comment c-style */
.c-test.b=2
// C++-style comment
.cpp-test={}
// Multiline comment cpp-style
// Second line
.cpp-test.c=3
// Comment before double
.cpp-test.d=4.1
// Comment before string
.cpp-test.e="e-string"
// Comment before true
.cpp-test.f=true
// Comment before false
.cpp-test.g=false
// Comment before null
.cpp-test.h=null
